globus_action_provider_tools/authentication.py

```diff
@@ -176,17 +213,37 @@ def get_authorizer_for_scope(
         token_expiration = dep_tkn_resp.get("expires_at_seconds")
```

Comment:
`token_expiration` has the possibility to be `None`, and can still raise a `TypeError` in the code below when compared to `now`. Would it make sense to use a default of `0` so that it will be the correct type and can still fail the inequality comparison correctly?

```python
token_expiration = dep_tkn_resp.get("expires_at_seconds", 0)
```

Reply:
I was looking at an out-of-date version. I'm deleting this comment. Sorry for the confusion!